src/modules/roles/application/use-cases/__tests__/update-user-roles.use-case.spec.ts

```
@@ -66,13 +70,20 @@
         roles: [existingRole, newRole],
       });
 
-      const result = await useCase.execute(userId, roleId, undefined, currentUserId);
+      const result = await useCase.execute(
```

ESLint

[error] 'result' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)